bl32/tsp/aarch64/tsp_entrypoint.S

```diff
@@ -43,10 +43,7 @@
 	msr	spsr_el1, \reg2
 	.endm
 
-	.section	.text, "ax"
-	.align 3
-
-func tsp_entrypoint
+func tsp_entrypoint _align=3
```

Comment:
I suspect the .align 3 here is a historical artefact and is not needed. But that's off-topic.